jakttest/run-all.sh

```diff
@@ -5,13 +5,23 @@ if [ ! -x ./build/jakttest ]; then
     cargo run jakttest/jakttest.jakt
 fi
 
+# check for jakttest/ mtime and build/jakttest mtime
+{
+    jakttest_binary_mtime=$(stat -c %Y build/jakttest)
+    latest_jakttest_source_mtime=$(stat -c %Y jakttest/*.jakt | sort | tail -n1)
```

Comment:
Maybe it would be better to have a separate function since we're reusing the same cryptic `stat -c | sort | ...` pipeline

Reply:
I don't know if that abstraction would be worth it here, if it is done a 3rd time and the script grows a lot more (it should even shrink with #772 by quite a bit, at least in %), maybe ...